gffquant/profilers/feature_quantifier.py

```
@@ -349,8 +389,12 @@ def finalise(
                 round(self.aln_counter["filtered_read_count"] / self.aln_counter["full_read_count"], 3) * 100,
             )
 
+        self.adm.clear_caches()
         logger.info("Finished.")
 
+    def evaluate_alignment_group(self, aln_group, is_ambig_alignment):
+        ...
```

Ruff (0.8.0)

`FeatureQuantifier.evaluate_alignment_group` is an empty method in an abstract base class, but has no abstract decorator

(B027)